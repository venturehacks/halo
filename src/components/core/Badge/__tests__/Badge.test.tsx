import { render } from 'enzyme';
import React from 'react';

import { Badge } from '../index';

test('smoke', () => {
  const component = render(<Badge>NICE BADGE</Badge>);
  expect(component.text()).toEqual('NICE BADGE');
});
