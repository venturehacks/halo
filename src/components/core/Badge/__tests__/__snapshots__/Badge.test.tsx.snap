// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge snapshot color blue 1`] = `
<div
  class="component blue circle"
/>
`;

exports[`Badge snapshot color gray 1`] = `
<div
  class="component gray circle"
/>
`;

exports[`Badge snapshot color green 1`] = `
<div
  class="component green circle"
/>
`;

exports[`Badge snapshot shape circle 1`] = `
<div
  class="component gray circle"
/>
`;

exports[`Badge snapshot shape square 1`] = `
<div
  class="component gray square"
/>
`;
